app/views/gardens/show.html.erb

```diff
@@ -3,8 +3,13 @@
 <div class="plants">
   <ul>
     <% @garden.plants.each do |plant| %>
-      <li><%= plant.name %><br />
-      Watering Requirements: <%= plant.times_per_week.round(0) %> times/week</li>
+      <div id="plant-<%= plant.id %>"
+        <li>
+          <%= plant.name %><br />
+          Watering Requirements: <%= plant.times_per_week.round(0) %> times/week<br />
+          <%= button_to 'Remove Plant', plant_path(plant), method: :delete %>
```

Comment:
Here's an example of what I mean `button_to("Delete Item", dashboard_item_path(item), method: :delete, data: {confirm: "Are you sure?"})`

Reply:
Added, thanks!